ipt_connect/IPTdev/templates/IPTdev/team_detail.html

```diff
@@ -54,5 +54,13 @@ <h3>{% trans 'Penalties' %}:</h3>
 
     {% endfor %} {% endif %}
 
+    <h3>{% trans 'Bonus points' %}:</h3>
+    <p>
+    	{% if team.bonus_points %}
+    		{{team.bonus_points}}
```

Comment:
What would you think of breaking down the bonus points per rounds, i.e. something like

- 2 pts for winning round 1
- 1 pt for finishing 2nd on round 2
- etc... 

?

Reply:
To be honest, I'd prefer another approach. Firstly we should write a function which takes a round and gives us an object completely describing all 3 or 4 rounds of that fight, e.g. 2D-array (indexed with teams an rounds) of objects containing team roles and gathered points.
Such an array will be  useful both for summaries and bonus points.